### DISCUSSION TITLE
How to add version to app

### DISCUSSION THREAD

**Discussion by Moosems** [pyinstaller/pyinstaller#6921] [Help] [answered]
How do I give my macosx bundle a version such as `0.0.8`? I know it will likely be done in the spec file but when messing around with it I was unable to figure it out.

**Comment by jamesbcd** [comment, marked as answer]
For Mac, see https://pyinstaller.org/en/stable/spec-files.html#spec-file-options-for-a-macos-bundle

Here's an example:
```
app = BUNDLE(bundle_obj,
        name='MyApp.app',
        icon='relative/path/to/icon.icns',
        bundle_identifier="com.myapp",
        info_plist={
            'CFBundleShortVersionString': "1.2.3",
        }
```

For Windows, the situation is more complicated. You basically have to use an existing example as a template. See https://pyinstaller.org/en/stable/usage.html#capturing-windows-version-data
For my app I've used InnoSetup to give the installer a version, but haven't bothered giving the app itself a version.
